internal/pkg/cache/redis/redis.go

```diff
@@ -31,8 +33,25 @@ func NewRedis(address string, opts ...redis.DialOption) *Redis {
 		Dial: func() (redis.Conn, error) {
 			return redis.Dial("tcp", address, opts...)
 		},
-		MaxIdle:   500,
-		MaxActive: 10000,
+		MaxIdle:     200,
```

Comment:
Why do we need to change the default values here?
Also can we implement this as a wrapper of NewRedisWithConfig? I think the canonical values should be in service.yaml.

Reply:
My apologies, I did that to test and forgot to replace with default values.